theme/less/_navigation.less

```diff
@@ -6,21 +6,21 @@ nav
     {
       a
       {
-        &:hover
-        {
-          text-decoration: underline;
-        }
+        font-weight: 500;
+        font-size: 24px;
         text-decoration: none;
-        color: @reverse;
+        color: #626262;
       }
-      display: inline-block;
-      padding: 10px 10px 10px 0;
+      padding: 10px 45px 10px 0;
```

Comment:
Meeting notes: 
- Let's use paddings on `a` element.

Reply:
Instead of not equal paddings we can use this -> https://developer.mozilla.org/en-US/docs/Web/CSS/gap on `ul` element. But it's lack of support for Saffari.